policyengine_canada/variables/gov/cra/tax/income/credits/gst_credit/gst_credit_dependent_children.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_canada.model_api import *
+
+
+class gst_credit_dependent_children(Variable):
+    value_type = int
+    entity = Household
+    label = "GST credit dependent Children"
+    unit = CAD
+    documentation = "Number of eligible dependent children"
```

Comment:
At least this is how I've seen CA government sites spell it
```suggestion
    documentation = "Number of eligible dependant children"
```